src/components/app/App.jsx

```diff
@@ -1,5 +1,26 @@
 import React from 'react';
+import {
+  Router,
```

Comment:
You need to import BrowserRouter as Router - this could be why nothing is rendering